python/ql/src/experimental/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -131,18 +147,24 @@ module EssaFlow {
  * excludes SSA flow through instance fields.
  */
 predicate simpleLocalFlowStep(Node nodeFrom, Node nodeTo) {
-  not nodeFrom.(EssaNode).getVar() instanceof GlobalSsaVariable and
-  not nodeTo.(EssaNode).getVar() instanceof GlobalSsaVariable and
-  EssaFlow::essaFlowStep(update(nodeFrom), nodeTo)
+  exists(Node preUpdate |
+    not preUpdate.(EssaNode).getVar() instanceof GlobalSsaVariable and
+    not nodeTo.(EssaNode).getVar() instanceof GlobalSsaVariable and
+    EssaFlow::essaFlowStep(preUpdate, nodeTo) and
+    nodeFrom = update(preUpdate)
+  )
 }
 
+/**
+ * If a node `n` has a post-update node `pun(n)`, we want forward flow to go from
+ * `pun(n)` as well as from `n`.
+ */
```

Comment:
```suggestion
/**
 * Holds if `result` is either `node`, or the post-update node for `node`.
 */
```

Reply:
Yeah, the original comment should really be at the (singular) call site..